src/routes/CollectionsRoute.js

```diff
@@ -127,7 +128,12 @@ class CollectionsRoute extends React.Component {
   }
 
   render() {
-    const { location, match, children } = this.props;
+    const { location, match, children, stripes } = this.props;
+    const hasPerms = stripes.hasPerm('finc-select.metadata-collections.collection.get');
```

Comment:
Please dont use backend permissions. Use `ui-finc-select.view` instead.

Reply:
Thanks!
We should also use "ui-finc-select.all" instead of "finc-select.filters.item.put".
Here or in a new ticket?

Reply:
Let’s keep this PR focused and clean.